pkg/config/config.go

```diff
@@ -16,6 +17,22 @@ var (
 	DefaultCNIDir = "/var/lib/cni/sriov"
 )
 
+// SetLogging sets global logging parameters.
+func SetLogging(stdinData []byte, containerID, netns, ifName string) {
+	if logLevel, logFile, err := loadLoggingConf(stdinData); err == nil {
```

Comment:
I think that this is a valid error. If we are not able to unmarshall it, the plugin should return an error. What do you think?

Also, no need to have 2 separate functions. You can embed loadLoggingConf here. It's fine if you prefer to it separated.

Reply:
ok to both